src/Tickets/Emails/Admin/Preview_Modal.php

```diff
@@ -69,12 +71,10 @@ public function render_modal() {
 	 *
 	 * @since 5.5.7
 	 *
-	 * @param array $args Override default args by sending them in the `$args`.
-	 *
 	 * @return array The default modal args.
 	 */
-	public function get_modal_args( $args = [] ): array {
-		$default_args = [
+	public function get_modal_args(): array {
```

Comment:
Why are removing the ability to pass in custom arguments here?

Reply:
Currently there isn't any need for this to exist, and when it does what we would do here, is combine a `set_args()` with a couple of other changes to the Modal Class to avoid having a floating template editor instance be used to load the modal.

Ideally we would make `Preview_Modal` extend `Template__Template` from Common, which would enable us to avoid `get_modal_content()` to be so overloaded. Currently it does a little too much.